src/libp2p.js

```diff
@@ -6,6 +6,7 @@ const WS = require('libp2p-websockets')
 const Bootstrap = require('libp2p-bootstrap')
 const MPLEX = require('libp2p-mplex')
 const SECIO = require('libp2p-secio')
+const { Noise } = require('libp2p-noise')
```

Comment:
```suggestion
const {NOISE} = require('libp2p-noise')
```
This will create default noise instance

Reply:
@mpetrunic  our linting states that in these cases we must have a space between the `{` and the var name. In terms of end result, it is the same, it will create the default noise instance

Reply:
@mpetrunic  our linting states that in these cases we must have a space between the `{` and the var name. We need to keep it, or the lint job will fail

Reply:
@mpetrunic  our linting states that in these cases we must have a space between the `{` and the var name. We need to keep it, or the lint job will fail

Reply:
updated suggestion :+1: